docs/topics/gradle/gradle-configure-project.md

```diff
@@ -123,19 +128,66 @@ In the build module, you may have related compile tasks, for example:
 >
 {type="note"}
 
-For related tasks like these, the Kotlin Gradle plugin checks for JVM target compatibility. Different values of `jvmTarget` in
-the `kotlin` extension and [`targetCompatibility`](https://docs.gradle.org/current/userguide/java_plugin.html#sec:java-extension)
-in the `java` extension cause JVM target incompatibility. For example:
+For related tasks like these, the Kotlin Gradle plugin checks for JVM target compatibility. Different values of 
+the [`jvmTarget` attribute](gradle-compiler-options.md#attributes-specific-to-jvm) in the `kotlin` extension or task 
+and [`targetCompatibility`](https://docs.gradle.org/current/userguide/java_plugin.html#sec:java-extension)
+in the `java` extension or task cause JVM target incompatibility. For example:
 the `compileKotlin` task has `jvmTarget=1.8`, and
 the `compileJava` task has (or [inherits](https://docs.gradle.org/current/userguide/java_plugin.html#sec:java-extension)) `targetCompatibility=15`.
 
 Configure the behavior of this check by setting the `kotlin.jvm.target.validation.mode` property in the `build.gradle`
 file to:
 
-* `warning` – the default value; the Kotlin Gradle plugin will print a warning message.
-* `error` – the plugin will fail the build.
+* `error` – the plugin will fail the build; the default value for projects on Gradle 8.0+.
+* `warning` – the Kotlin Gradle plugin will print a warning message; the default value for projects on Gradle less than 8.0.
 * `ignore` – the plugin will skip the check and won't produce any messages.
 
+To avoid JVM target incompatibility, [configure a toolchain](#gradle-java-toolchains-support) or align JVM versions manually.
+
+#### What can go wrong if not checking targets compatibility {initial-collapse-state="collapsed"}
```

Comment:
```suggestion
#### What can go wrong if targets aren't checked for compatibility {initial-collapse-state="collapsed"}
```

Reply:
Do you think we need to use passive here?
Maybe "What can go wrong if you don't set up checking targets compatibility"?

Reply:
@sarahhaggarty ?